scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestControllerTest.java

```diff
@@ -114,15 +114,15 @@ public void testCreateAndGetSchedules() throws Exception {
 	@Test
 	public void testCreateSchedule_already_existing_schedule_for_appId() throws Exception {
 		String appId = TestDataSetupHelper.generateAppIds(1)[0];
-		// Create one specific date schedule for the application.
-		callCreateSchedules(appId, 1, 0);
+		// Create one specific date schedule and two recurring schedules for the application.
+		callCreateSchedules(appId, 1, 2);
 
-		// Create two specific date schedule for the same application.
-		ResultActions resultActions = callCreateSchedules(appId, 2, 0);
+		// Create two specific date schedules and one recurring schedule for the same application.
+		ResultActions resultActions = callCreateSchedules(appId, 2, 1);
```

Comment:
The naming is a bit poor since it is doing more than just Create, its doing an Upsert.